U/fe/1

frossie opened the pull request:
Had a first pass, this both serves as a delivery note for project management reasons and also fulfils DMLT action to write a note describing division of responsibilities between SQuaRE and CET for operations.

frossie committed:
Stub

frossie committed:
First draft

jonathansick committed:
Omit community_mailbot

We're not operating this service anymore; nobody has really complained about its absence, and I feel that the Slack notifications cover the in-project notification use cases well and the forum's own email notifications better service the public than mailing lists.

jonathansick committed:
Add 1Password

jonathansick reviewed (approved):
Looks good, thanks. I'll also mention that I have a technote, [SQR-043](https://sqr-043.lsst.io) in the works to serve as an operations guide. I haven't finished even the basic content so it's probably not worth linking to it at this stage.

frossie commented:
> Looks good, thanks. I'll also mention that I have a technote, [SQR-043](https://sqr-043.lsst.io) in the works to serve as an operations guide. I haven't finished even the basic content so it's probably not worth linking to it at this stage.

Yeah I saw that but since there was nothing pushed I decided to leave it out.

frossie committed:
Processed suggestions by Wil

Wil suggested adding a Known Issues section, and also we agreed that like an LDM a Delivery Note is accepted when it is merged to master upon review.

frossie committed:
Minor edits
